handlers/schemas/UploadVOD.yaml

```diff
@@ -2,7 +2,6 @@ type: "object"
 properties:
   url:
     type: "string"
-    format: "uri"
```

Comment:
This is so that we can use local file paths in testing

Reply:
TODO: Check ObjectStore and see if we can use a file:// syntax to avoid changing this

Reply:
Wouldn't this allow end-users to pass a local file using file:// to the studio api?

Reply:
Studio rejects it 
```
{
  "errors": [
    "{\"keyword\":\"pattern\",\"dataPath\":\".url\",\"schemaPath\":\"#/properties/url/pattern\",\"params\":{\"pattern\":\"^(https?|ipfs|ar)://\"},\"message\":\"should match pattern \\\"^(https?|ipfs|ar)://\\\"\"}"
  ]
}
```

Reply:
`file://` seems to work on our side though, so I've updated the tests and added this validation back in